### ISSUE TITLE
Missing alt texts for collection images

### ISSUE THREAD

**Issue by renanvsouza** [Shopify/dawn#3234]
## Describe the current behavior

The alt text for the collection featured image is not being pulled in the collection template and collection card.

## Describe the expected behavior

The theme should pull the alt text defined in the image.

## Version information (Dawn, browsers and operating systems)

- Dawn Version: 13.0.0

## Possible solution

Adding Liquid tags to pull these alt texts.

## Additional context/screenshots
sections/main-collection-banner.liquid
![screenshot1](https://github.com/Shopify/dawn/assets/77407058/9c9c315d-08ac-4ffb-b91e-c38da628cc83)

snippets/card-collection.liquid
![screenshot2](https://github.com/Shopify/dawn/assets/77407058/20ff8ef6-1c4e-4e90-9c00-b22e327bc355)
